l1-contracts-foundry/script/InitializeL2WethToken.s.sol

```diff
@@ -41,29 +41,17 @@ contract InitializeL2WethTokenScript is Script {
     function initializeConfig() internal {
         config.deployerAddress = msg.sender;
 
-        // Parse some config from output of l1 deployment
         string memory root = vm.projectRoot();
-        string memory path = string.concat(root, "/script-out/output-deploy-l1.toml");
+        string memory path = string.concat(root, "/script-config/config-initialize-l2-weth-token.toml");
         string memory toml = vm.readFile(path);
 
+        config.l1WethTokenAddr = toml.readAddress("$.l1_weth_token_addr");
+        config.l1WethTokenName = toml.readString("$.l1_weth_token_name");
+        config.l1WethTokenSymbol = toml.readString("$.l1_weth_token_symbol");
```

Comment:
could we hardcode it? looks like we can't change name and symbol of weth..

Reply:
Yes, we can hardcode it.